### DISCUSSION TITLE
How to edit name with chinese at profile in Account Details?

### DISCUSSION THREAD

**Discussion by weijuer** [dailydotdev/daily#522] [Feature Requests]
As you can see, I can't enter chinese chars here.
![Snipaste_2022-04-17_00-16-32](https://user-images.githubusercontent.com/12313268/163683045-1765d3b6-6f27-4c92-a991-6294c0f2ac05.png)

And also, I can't save changes with the same email either.
![image](https://user-images.githubusercontent.com/12313268/163683160-be01a7fa-6be4-4195-8128-65b20c78710a.png)

**Comment by idoshamun** [comment]
Hi,

It should work now. Please confirm
